### DISCUSSION TITLE
Add FanEntityFeature.TURN_OFF / FanEntityFeature.TURN_ON

### DISCUSSION THREAD

**Discussion by CFenner** [home-assistant/architecture#1097] [Entity Models] [answered]
With https://github.com/home-assistant/core/pull/114175 I intended to add a fan entity to the ViCare integration that is an always-on device which cannot be turned off. 

For that it would be good to have an `FanEntityFeature.TURN_OFF` and `FanEntityFeature.TURN_ON` flag in the fan entity.

**Comment by frenck** [comment, marked as answer]
We have discussed this one during our core team meeting, and agree this can be done.
This will make it consistent with other similar platforms as well.

Feel free to continue on this implementation.

../Frenck

**Reply by gjohansson-ST** [reply]
Implementation is being done in https://github.com/home-assistant/core/pull/121447

**Comment by emontnemery** [comment]
I'm closing this since the discussion is answered and there's a merged implementation PR.
